mobilecoind/api/proto/mobilecoind_api.proto

```diff
@@ -158,9 +160,45 @@ message UnspentTxOut {
     // The token id of the TxOut
     uint64 token_id = 7;
 
+    // The (decrypted) MemoPayload of this TxOut
+    bytes memo_payload = 8;
+
     // The monitor id this UnspentTxOut belongs to.
     // Note that this field is not included in the Rust `utxo_store::UnspentTxOut` struct.
     bytes monitor_id = 10;
+
+    // The decoded memo info, if any
+    DecodedMemo decoded_memo = 11;
```

Comment:
@eranrund do you know why that gap happened? i don't remember reviewing that change

Reply:
@nick-mobilecoin my guess is like, at one point, the `UnspentTxOut` proto in `mobilecoind_api.proto` was exactly what is stored in the UTXOs lmdb db. But later, we decided to add `monitor_id` to the API response, or something like this, to make the API easier to use. But we did not want to add the `monitor_id` to every UTXO record in the database, because that would bloat the database and in that context, the `monitor_id` of the UTXO is always clear so it would be wasteful. So at this point these two protos diverged.

I think `monitor_id` was assigned an index of 10 because it was somewhat larger than the other existing indices at the time, and this separates the "API only" from the "model & API" fields.

Now, we've had so many model & API fields added that we're getting to close to 10.

I added `decoded_memo` at 11 because it its an "API only" and not a "model & API" field, so I was trying to keep that organization going.

I don't know if that was actually intentional or if I imagined this though.

Reply:
@cbeck88 I think that's exactly the reason. Its also called out in the comment here https://github.com/mobilecoinfoundation/mobilecoin/blob/main/mobilecoind/api/proto/mobilecoind_api.proto#L156

Reply:
i'm not sure it's still important to align these numbers, if they are different protos, they are different now, right?

maybe there was some concern of compatibility a long time ago, but I don't see that there is anywhere that we are taking bytes from the DB and interpreting them using the mobilecoind_api.proto UnspentTxOut.

Reply:
yeah i don't think it matters